### DISCUSSION TITLE
learned from my mistake

### DISCUSSION THREAD

**Discussion by sdrowkcabrepyt** [community/community#125179] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hey mate its Ian waters again im sorry for what i did, can i please be let back into the discord and regain access to the mod?
![Uploading image_2024-05-22_220000005.png…]()

**Comment by ebndev** [comment]
Hi @sdrowkcabrepyt,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a Discord server or specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
